app/views/components/_countdown.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if clock.show? %>
+  <%= clock.days_left %><%= clock.days_text %> to go
```

Comment:
I'm assuming this is English only for now while we're doing the test.

Reply:
Yes that's right. We haven't been provided with the `cy` translation of the content, and we're only testing in the `en` locale. But if the AB test shows the clock is a good thing, we would at that point need to have a translation which we might forget. I'll mention to Nick so we don't.